lib/msgServer/msgStream/transports/http-polling/index.js

```diff
@@ -105,7 +105,7 @@ HttpPollingHost.prototype.deliver = function (msgs) {
 
 
 HttpPollingHost.prototype.respondBadRequest = function (reason) {
-	if (!this.res) {
+	if (this.res.finished) {
```

Comment:
Yeah, that case doesn't worry me. It was the after-send case that had me worried. But as you point out below, sounds pretty safe. I hope it turns out to be as consistent and predictable as we would need it to be.